MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include forte *
```

Comment:
This change also adds CoNLL eval script to forte library. Since it is a perl script, by default, python packaging does not copy perl scripts into build folder. Through this manifest file, I am saying to recursively copy all the files in the `forte` while packaging the `forte` library.

Reply:
Interesting, thank you.